pillarbox-cast/src/main/java/ch/srgssr/pillarbox/cast/PillarboxCastPlayer.kt

```diff
@@ -304,15 +326,17 @@ class PillarboxCastPlayer internal constructor(
 
     private inner class MediaQueueCallback : MediaQueue.Callback()
 
-    companion object {
+    private companion object {
         private const val TAG = "CastSimplePlayer"
         private val AVAILABLE_COMMAND = Player.Commands.Builder()
             .addAll(
                 COMMAND_PLAY_PAUSE,
                 COMMAND_GET_CURRENT_MEDIA_ITEM,
                 COMMAND_GET_TIMELINE,
                 COMMAND_STOP,
-                COMMAND_RELEASE
+                COMMAND_RELEASE,
+                COMMAND_SET_SHUFFLE_MODE,
```

Comment:
I think shuffle mode is not always available and it can be fetch with 

```kotlin
remoteMediaClient?.mediaStatus?.isMediaCommandSupported(MediaStatus.COMMAND_QUEUE_SHUFFLE)
```

Reply:
Shuffle seem to work even if the command is flagged has not available.